### ISSUE TITLE
Run agent in Synology DSM

### ISSUE THREAD

**Issue by iBobik** [tutumcloud/tutum-agent#33]
We use Synology server also as a development server, because from some date they officially support Docker and has nice GUI for it. But this UI is nothing compared to Tutum UI :-)

So how to "bring my own Synology node"? Their Linux is not on the list of supported distributions, so it will propably needs some hacking.

Propably just conect Tutum Agent with host's Docker deamon? Run agent in container and configure it to use socketfile of host?

**Comment by matiasinsaurralde** [comment]
Do you have shell access to your Synology? Which distro are they based on?

**Comment by iBobik** [comment]
Yes, root shell si officialy supported. Synology DSM is distro, not seems it is based on some other common distro. But it has Linux kernel and full featured Docker client and daemon.

Jan Pobořil
http://honza.poboril.cz/

22. 11. 2015 v 8:07, Matias Insaurralde <notifications@github.com>:
